xlib/main.c

```diff
@@ -884,19 +882,58 @@ static int systemlang(void)
     return ui_guess_lang_by_posix_locale(str, DEFAULT_LANG);
 }
 
+_Bool parse_args_wait_for_theme;
+char theme;
+
 int main(int argc, char *argv[])
 {
-    if(argc == 2 && argv[1]) {
-        if(!strcmp(argv[1], "--version")) {
-            debug("%s\n", VERSION);
-            return 0;
-        } else if(!strcmp(argv[1], "--portable")) {
-            debug("Launching uTox in portable mode: All data will be saved to the tox folder in the current working directory\n");
-            utox_portable = 1;
-        } else {
-            debug("Valid arguments are: --version and --portable (launches uTox in portable mode)\n");
-            return 0;
+    parse_args_wait_for_theme = 0;
```

Comment:
`argv[i+1]` can easily lead to an overflow if this is the last argument. If I am going to remove the 'parse_args_wait_for_theme', will need to add extra check for array length.